src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +45,32 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, dateTimeFormatter);
+        LocalDate localDateTo = LocalDate.parse(dateTo, dateTimeFormatter);
+        if (localDateFrom.compareTo(localDateTo) >= 0) {
```

Comment:
try to solve without `compareTo` at all. there are more relevant methods

Reply:
changed them to `isAfter`, `isEqual` and `isBefore` methods